### PR TITLE
Traciblity

### PR DESCRIPTION
**Pull request by johandahlberg**
Adding a `trace_runfolder.py` script, which can be used to trace all executions for a tag. Basically a utility wrapper for a few of the StackStorm API, but it should make things a lot smoother for the operator.

### CODE REVIEW

File: actions/workflows/ngi_uu_workflow.yaml [hunk 1]
```diff
@@ -23,6 +23,14 @@ workflows:
 
         tasks:
             ### GENERAL TASKS START ###
+            note_workflow_hash:
```
Comment (withrocks) [inline actions/workflows/ngi_uu_workflow.yaml]: I don't follow what's happening here.

File: scripts/trace_runfolder.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+def get_executions_from_traces(traces, headers):
+    executions = []
+    for trace in traces:
+        trace_id = trace["id"]
+        trace_info_response = requests.get(
+            "{}/{}/{}".format(api_base_url, "traces", trace_id),
+            headers=headers)
+        trace_info = json.loads(trace_info_response.text)
+        action_executions = map(lambda x: x["object_id"], trace_info["action_executions"])
+        executions += action_executions
```
Comment (withrocks) [inline scripts/trace_runfolder.py]: Personally I would yield to save 2-3 lines :P - but that's a matter of taste

File: scripts/trace_runfolder.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    return actions
+
+def filter_actions_by_name(actions, name):
+    return [action for action in actions if action["action"]["name"] == name]
+
+# -----------
+# Main script
+# -----------
+traces = get_traces_for_tag(args.tag, access_headers)
```
Comment (withrocks) [inline scripts/trace_runfolder.py]: Wouldn't hurt to put this into a __main__ block, if you would want to use this as a library. But that can wait until then.

File: actions/workflows/ngi_uu_workflow.yaml (hunk 1 above)
Comment (withrocks) [inline actions/workflows/ngi_uu_workflow.yaml]: :+1: Now I get it - the name of the task was confusing.

**Comment by withrocks** [comment]
Looks good! Tell me when you want it merged.

**Comment by johandahlberg** [comment]
Great comments! This should be good to merge now.

**Pull request merged**